projects/packages/videopress/src/client/admin/components/clipboard-button-input/index.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@automattic/jetpack-components';
+import { useCopyToClipboard } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+import { useState } from 'react';
+/**
+ * Internal dependencies
+ */
+import styles from './style.module.scss';
+import { ClipboardButtonInput } from './types';
+import type React from 'react';
+
+const noop = () => {
+	/* noop */
+};
+
+/**
+ * Video Details Card component
+ *
+ * @param {ClipboardButtonInput} props - Component props.
+ * @returns {React.ReactNode} - VideoDetailsCard react component.
+ */
+const VideoDetailsCard: React.FC< ClipboardButtonInput > = ( {
```

Comment:
I started to see some points about not using `React.FC`, I think we can take a look on it after.

https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components
https://github.com/facebook/create-react-app/pull/8177

Reply:
Let me fix it in a follow-up, please